src/__tests__/pet.test.js

```diff
@@ -0,0 +1,168 @@
+const { Pet } = require ('../pet.js');
+
+const ERROR_MESSAGE = 'Your pet is no longer alive :(';
+
+let fido;
```

Comment:
Good use of let and beforeEach here.

Reply:
To make your tests really robust, I would create a second variable here and assign it to a new pet so that you have two unique pets to test against to ensure nothing is hardcoded.